Agents/MetOfficeAgent/resources/metoffice.properties

```diff
@@ -0,0 +1,34 @@
+###--- Properties for MetOffice Agent agent ---###
```

Comment:
There should be a wider discussion about properties files as it should be possible to populate most of the information when starting the agent's container. What's here is fine for now though.